backend/tests/unit/core/schema/schema_branch/test_validate_computed_attributes.py

```diff
@@ -0,0 +1,266 @@
+import pytest
+
+from infrahub.core.constants import ComputedAttributeKind, RelationshipCardinality
+from infrahub.core.schema import AttributeSchema, GenericSchema, NodeSchema, RelationshipSchema, SchemaRoot
+from infrahub.core.schema.computed_attribute import ComputedAttribute
+from infrahub.core.schema.schema_branch import SchemaBranch
+
+
+@pytest.mark.parametrize(
+    "schema_root,expected_error",
+    [
+        pytest.param(
+            SchemaRoot(
```

Comment:
Could we create some variables for each param before the function
it feels like it would make it easier to read the file and understand what is going on

Reply:
Do you have some sort of example in mind? Each test case takes a SchemaRoot object and a string with the expected error. I can move everything out of the this scope and define it above in a different format. But I'm not sure how much it will help?

Reply:
not a blocker, personally I find it hard to read this way but we can revisit that later